notebooks/community/neo4j/graph_paysim.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<p>Please consider additional cleanup steps for all assets:</p><p><code></code><code></p><p>endpoint.undeploy_all()</p><p>endpoint.delete()</p><p>model.delete()</p><p>dataset.delete()</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
I think that'd be a good idea.  Would it be ok to get this PR in and then work with @htappen on that?

Reply:
Sure, we can defer this until after the initial PR gets merged.